tcms/core/tests/test_admin.py

```diff
@@ -44,21 +43,22 @@ def test_admin_display(self):
 
         self.assertNotContains(response, 'Test case status')
         self.assertNotContains(response, 'Test case texts')
-        self.assertNotContains(response, 'Test cases')
+        self.assertContains(response, 'Search Test Cases')
 
         # for tcms.testplans
         self.assertContains(response, 'Plan types')
 
-        self.assertNotContains(response, 'Test plans')
+        self.assertContains(response, 'Search Test Plans')
 
         # for tcms.testruns
-        self.assertNotContains(response, 'Testruns')
-        self.assertNotContains(response, 'Test case run statuss')
+        self.assertNotContains(response, 'Test case run status')
         self.assertNotContains(response, 'Test case run statuses')
```

Comment:
This is now named `TestExecutionStatus` model and it is not currently shown in admin. Once @asankov finishes refactoring of this part we'll have to show these fields again.

Try registering an admin section for the model to see which would be the appropriate name to assert that it is not shown.